backend/infrahub/testing/container.py

```diff
@@ -0,0 +1,108 @@
+import os
+import uuid
+from dataclasses import dataclass
+from functools import cached_property
+from pathlib import Path
+from typing import Optional
+
+from pydantic import BaseModel
+from testcontainers.compose import DockerCompose
+from typing_extensions import Self
+
+from infrahub import __version__ as infrahub_version
+
+
+class ContainerService(BaseModel):
+    container: str
+    port: int
+
+
+INFRAHUB_SERVICES: dict[str, ContainerService] = {
+    "server": ContainerService(container="infrahub-server", port=8000),
+    "task-manager": ContainerService(container="task-manager", port=4200),
```

Comment:
I think ports should be defined as global constants here as they are reused below

Reply:
I'll revisit that in the next PR,
technically these ports are "fixed" since there are the internal port numbers, not the external one.